### ISSUE TITLE
Apply rule to display fields on upload page depending on the product type

### ISSUE THREAD

**Issue by Gschwend** [linea-it/pz-server#72]
Rules: 

| Product type | metadata |
|---|---|
| **Spec-z Catalog**| Survey, Description |
| **Training Set**| Release, Description|
| **Validation Set**| Release, Description|
| **Validation Results**| Release, PZ Code, Description|
| **Photo-z Table**| Release, PZ Code, Description|

Basically, to fix the current implementation, just:
 
* Remove **Release** from the Spec-z Catalog
* Add **PZ Code** to Photo-z Table
